packages/@expo/config-plugins/src/utils/locales.ts

```diff
@@ -0,0 +1,50 @@
+import { ExpoConfig } from '@expo/config-types';
+import JsonFile, { JSONObject } from '@expo/json-file';
+import { join } from 'path';
```

Comment:
```suggestion
import path from 'path';
```

nit: prefer use `path.join()` in code (though i know the code was coming from previous ios Locale.ts)